app/assets/stylesheets/pageflow/editor/loading.css.scss

```diff
@@ -0,0 +1,31 @@
+.spinner > div {
+  background-color: #ddd;
```

Comment:
Color literals like `#ddd` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets